smart_fact_crawler/tools.py

```diff
@@ -23,10 +23,7 @@ def smartfact_time2datetime(fact_timestamp):
 
 
 def parse_table(text):
-    return [
-        line.split('\t')
-        for line in html.unescape(text).splitlines()
```

Comment:
was the html.unescape() removed on purpose? Or was it lost accidentally while rearranging the line?

Reply:
Ah, I see. It is removed on purpose.